src/modules/claim/claim.service.ts

```diff
@@ -42,20 +38,18 @@ export class ClaimService {
     private readonly roleClaimRepository: Repository<RoleClaim>,
     @InjectRepository(Claim)
     private readonly claimRepository: Repository<Claim>,
-    private readonly assetService: AssetsService,
-    @InjectQueue('claims') private readonly claimQueue: Queue<string>,
-    private readonly nats: NatsService,
+    private readonly assetService: AssetsService, //@InjectQueue('claims') private readonly claimQueue: Queue<string>, //private readonly nats: NatsService,
   ) {
     this.logger.setContext(ClaimService.name);
-    this.initializeExchangeListener();
+    //this.initializeExchangeListener();
   }
 
-  private initializeExchangeListener() {
-    this.nats.connection.subscribe(`*.${NATS_EXCHANGE_TOPIC}`, async data => {
-      this.logger.debug(`Got message ${data}`);
-      await this.claimQueue.add('save', data);
-    });
-  }
+  // private initializeExchangeListener() {
+  //   this.nats.connection.subscribe(`*.${NATS_EXCHANGE_TOPIC}`, async data => {
```

Comment:
Please check https://github.com/energywebfoundation/iam-cache-server/pull/206
Add the same options for scaling purposes